config/config.go

```diff
@@ -24,8 +24,7 @@ type GRPCConfig struct {
 //Config Server config
 type Config struct {
 	Port string `default:"8080"`
-	//Timeout represents graceful shutdown period.
-	//Default is 60 seconds.
+	// Timeout represents graceful shutdown period. Defaults to 60 seconds.
 	Timeout  time.Duration `default:"60s"`
```

Comment:
Should we call it `ShutdownTimeout` or `Deadline` then? `Timeout` looks pretty generic?

Reply:
Yes. +1.  Will change in another PR. Avoided config name changes to limit the scope of PR. Planning to scope server configs under `server` as well. Which will allow us to provide client config and others as well.